server/src/org/apache/cloudstack/network/lb/CertServiceImpl.java

```diff
@@ -339,190 +348,182 @@ public SslCertResponse createCertResponse(SslCertVO cert, List<LoadBalancerCertM
         return response;
     }
 
-    private void validateCert(Certificate cert, boolean chainPresent) {
+    private void validateCert(final Certificate cert, final boolean chainPresent) {
 
-        if (!(cert instanceof X509Certificate))
+        if (!(cert instanceof X509Certificate)) {
             throw new IllegalArgumentException("Invalid certificate format. Expected X509 certificate");
+        }
```

Comment:
Please consider converting this `if` block to a `Preconditions.checkArgument` to DRY out the code.  Also, check that `cert` is not `null` to avoid a potential NPE.